packages/pigeon/platform_tests/test_plugin/windows/test/null_fields_test.cpp

```diff
@@ -154,28 +133,36 @@ TEST_F(NullFieldsTest, ReplyFromMapWithNulls) {
   EXPECT_EQ(reply.type(), nullptr);
 }
 
-TEST_F(NullFieldsTest, RequestToMapWithValues) {
+TEST_F(NullFieldsTest, RequestToListWithValues) {
   NullFieldsSearchRequest request;
   request.set_query("hello");
   request.set_identifier(1);
 
-  EncodableMap map = MapFromRequest(request);
+  EncodableList wrapped = ListFromRequest(request);
 
-  EXPECT_EQ(map.size(), 2);
-  EXPECT_EQ(*ExpectAndGet<std::string>(map, "query"), "hello");
-  EXPECT_EQ(*ExpectAndGet<int64_t>(map, "identifier"), 1);
+  EXPECT_EQ(wrapped.size(), 1);
+
+  EncodableList list = wrapped[0];
+  EXPECT_EQ(list.size(), 2);
+
+  EXPECT_EQ(list[0], "hello");
+  EXPECT_EQ(list[1], 1);
 }
 
 TEST_F(NullFieldsTest, RequestToMapWithNulls) {
   NullFieldsSearchRequest request;
   // TODO(gaaclarke): This needs a way to be enforced.
   request.set_identifier(1);
 
-  EncodableMap map = MapFromRequest(request);
+  EncodableList wrapped = ListFromRequest(request);
+
+  EXPECT_EQ(wrapped.size(), 2);
 
-  EXPECT_EQ(map.size(), 2);
-  EXPECT_TRUE(map[EncodableValue("hello")].IsNull());
-  EXPECT_EQ(*ExpectAndGet<int64_t>(map, "identifier"), 1);
+  EncodableList list = wrapped[0]
+
+      EXPECT_EQ(list.size(), 2);
```

Comment:
What's up with this funky formatting, missing a semicolon?  Weird it would compile.

Reply:
c++ formatter does strange things when semicolons are missing. Good catch on the semi